src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -46,9 +43,120 @@ void IHybridWebView.RawMessageReceived(string rawMessage)
 		/// </summary>
 		public event EventHandler<HybridWebViewRawMessageReceivedEventArgs>? RawMessageReceived;
 
+		/// <summary>
+		/// Sends a raw message to the code running in the web view. Raw messages have no additional processing.
+		/// </summary>
+		/// <param name="rawMessage"></param>
 		public void SendRawMessage(string rawMessage)
 		{
-			Handler?.Invoke(nameof(IHybridWebView.SendRawMessage), rawMessage);
+			Handler?.Invoke(
+				nameof(IHybridWebView.SendRawMessage),
+				new HybridWebViewRawMessage
```

Comment:
@mattleibow - this is the follow-up to a comment you had on an earlier PR. I added this 'container type' for better future-proofing.